app/domain/authentication/authn_oidc/authenticate.rb

```diff
@@ -13,6 +13,7 @@ module AuthnOidc
         enabled_authenticators:     ENV['CONJUR_AUTHENTICATORS'],
         fetch_oidc_secrets:         AuthnOidc::Util::FetchOidcSecrets.new,
         token_factory:              TokenFactory.new,
+        validate_account_exists:    ::Authentication::Security::ValidateAccountExists.new,
```

Comment:
Beyond the scope of this PR but, I see that inside the actual `validate_account_exists.rb` file, there is:
```
      def account_admin_role
        @role_class["#{@account}:user:admin"]
```
Why do we want to check the role of the account is admin? It can be any type of user no?

Reply:
we don't check that the role of the account is an admin. We check that there is a user admin in the given account. This is the way we check that an account exists as when an account is created, an admin user is created in the DB and that's it. So an account exists if and only if there is a user `<account>:user:admin`

Reply:
oh that's right. There needs to be at least 1 admin for an acc to exist